From a526b0088191d265c1ea703a45bbc5a4c5f4c429 Mon Sep 17 00:00:00 2001
From: Simon Sasburg <Simon.Sasburg@gmail.com>
Date: Wed, 26 Sep 2007 15:02:07 +0200
Subject: [PATCH] Fix link error regarding h_errno

It was being redeclared with a wrong linkage, this is now prevented by checking if we are building for msys.
---
 pp_sys.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/pp_sys.c b/pp_sys.c
index 8423bd0..76028c2 100644
--- a/pp_sys.c
+++ b/pp_sys.c
@@ -62,7 +62,7 @@ extern "C" int syscall(unsigned long,...);
    compiling multithreaded and singlethreaded ($ccflags et al).
    HOST_NOT_FOUND is typically defined in <netdb.h>.
 */
-#if defined(HOST_NOT_FOUND) && !defined(h_errno) && !defined(__CYGWIN__)
+#if defined(HOST_NOT_FOUND) && !defined(h_errno) && !defined(__CYGWIN__) && !defined(__MSYS__)
 extern int h_errno;
 #endif
 
-- 
1.5.3.GIT

